python/cudf/cudf/tests/test_indexing.py

```diff
@@ -889,6 +889,19 @@ def test_out_of_bounds_indexing():
         a[[0, 1, -4]] = 2
 
 
+def test_sliced_indexing():
```

Comment:
Could you add a test against a non-default index as well?

Reply:
This a non-default index, I am setting column `a` as index.